napari/components/viewer_model.py

```diff
@@ -658,9 +663,54 @@ def add_image(
 
             return layer_list
 
+    def open_sample(self, plugin: str, sample: str, **kwargs):
+        """Open `sample` from `plugin` and add it to the viewer.
+
+        To see all available samples registered by plugins, use
+        :func:`napari.plugins.available_samples`
+
+        Parameters
+        ----------
+        plugin : str
+            name of a plugin providing a sample
+        sample : str
+            name of the sample
+
+        Raises
+        ------
+        KeyError
+            If `plugin` does not provide a sample named `sample`.
+        """
+        from ..plugins import _sample_data, available_samples
+
+        try:
+            data = _sample_data[plugin][sample]
+        except KeyError:
+            samples = available_samples()
+            msg = (
```

Comment:
We need to localize these strings @tlambert03

Reply:
sorry!  it's going to take me a little while to remember that :)  keep at me

Reply:
I have a massive PR with ALL the strings in napari to translate, so soon we will get some CI failures to remind all of us :)